packages/esm-ward-app/src/ward-patient-card/ward-patient-card.component.tsx

```diff
@@ -32,6 +34,12 @@ const WardPatientCard: WardPatientCard = (wardPatient) => {
         ))}
         <ExtensionSlot name={headerExtensionSlotName} state={wardPatient} />
       </div>
+      {wardPatient?.inpatientRequest ? (
+        <div className={styles.wardPatientCardPendingItemsRow}>
+          <Hourglass className={styles.hourGlassIcon} size="24" />:
+          <WardPatientTransfer wardPatient={wardPatient} />
+        </div>
+      ) : null}
```

Comment:
@chibongho  thoughts on whether we can hard code this into the card or do we need/should we make this configurable?

Reply:
Offhand, that seems reasonable @brandones

Reply:
Yeah, I'm ok with making it not configurable until we have a use case that requires it.